main.tex

```diff
@@ -225,47 +497,162 @@ \subsubsection{On-ChainCircuit Breaker and the Breaker Box}
+  \item \textbf{Trove selection}\,:  When users redeem stablecoins for collateral, v2 targets troves with the \emph{lowest} interest rate first, effectively encouraging borrowers to keep rates competitive.
+  \item \textbf{Redemption fee}\,:  The fee is paid in collateral and added \emph{directly} to the redeemed trove rather than to veMENTO stakers.  This compensates the borrower for the forced deleveraging and preserves peg stability.
 \end{itemize}
 
-\subsubsection{Challenges}
+\subsubsection*{FPMM Liquidity Provider Incentives}
 \begin{itemize}
-    \item Fragmented Liquidity: The ease and affordability of launching new decentralized stablecoins pose challenges, such as the potential for multiple parties to introduce multiple versions of the same asset, leading to fragmented liquidity. To mitigate this, Mento will implement an incentivization module, enabling operators and users to incentivize borrowers and liquidity providers. Through the Gauge System, holders of MENTO governance tokens can direct a portion of future token emissions to specific asset stability pools or liquidity pools, fostering liquidity for meaningful assets while rendering less favored 
-    assets irrelevant.
-    \item Safeguarding Users: Ensuring end-user protection against stablecoins launched with questionable economic setups or malicious intent is paramount. While operators have the flexibility to select any collateral asset, the requirement of a price feed from data providers like RedStone limits the assets eligible for use as collateral. Additionally, the Gauge System acts as a second line of defense, allowing the community to review and reject assets lacking sound economic fundamentals through governance decisions.
-    \item Governance and Revenue Sharing: The fee income generated by the Stablecoin Factory is expected to be allocated between the Mento treasury, Mento contributors and MENTO token holders participating in the governance process. Token holders can vote on specific stablecoin pools, ensuring community involvement in decision-making processes.
+  \item \textbf{Spread capture}\,:  FPMM LPs quote a fixed bid/ask around the oracle price and earn the embedded spread on each user swap.
+  \item \textbf{MENTO emissions via gauges}\,:  A governor\hyp controlled ``gauge'' streams a capped USD value of MENTO tokens to staked LP tokens, ensuring competitive on\hyp chain liquidity without overpaying.
+  \item \textbf{Swap fee (\texttt{amm\_fee})}\,:  Every trade pays an additional micro\hyp fee that flows \emph{entirely} to LPs during normal, two\hyp sided volume.
```

Comment:
why do we highlight _entirely_ here? as it stands it reads as if all three components (spread / mentor emissions / swap fee) go entirely to LPs